shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -69,6 +101,14 @@ protected int getNumOfArgs() {
 
   @Override
   public int run(CommandLine cl) throws IOException {
+    String[] args = cl.getArgs();
+    if (cl.hasOption("h")
+        && !(args.length > 0 && args[0].equalsIgnoreCase("CAPACITY"))) {
```

Comment:
using `equalsIgnoreCase` can lead to confusion. Some people will use "Capacity", others will use "capacity", and others will use "CAPACITY". Nobody will know which one is the right one to use. shells are not normally case sensitive

Reply:
Changed to do not support any uppercase category.
I will change my favor to support all the word cases.
Thanks!

```java
    if (cl.hasOption("h")
        && !(args.length > 0 && args[0].equals("capacity"))) {
      // if category is capacity, we print report capacity usage inside CapacityCommand.
      System.out.println(getUsage());
      System.out.println(getDescription());
      return 0;
    }

    // Get the report category
    Command command = Command.SUMMARY;
    if (args.length == 1) {
      switch (args[0]) {
        case "summary":
          command = Command.SUMMARY;
          break;
        case "capacity":
          command = Command.CAPACITY;
          break;
        default:
          throw new InvalidArgumentException("report category is invalid.");
      }
    }
```